setup.cfg

```
@@ -41,29 +41,21 @@ exclude =
 
 [options.extras_require]
 testing =
+	setuptools[testing-integration]
+
 	# upstream
-	pytest >= 6
 	pytest-checkdocs >= 2.4
 	pytest-cov; \
 		# coverage seems to make PyPy extremely slow
 		python_implementation != "PyPy"
 	pytest-mypy >= 0.9.1; \
-		# workaround for jaraco/skeleton#22
+		# workaround for https://github.com/jaraco/skeleton/issues/22
```

```suggestion
		# workaround for jaraco/skeleton#22
```
This line didn't need to change